### ISSUE TITLE
False Negative - PHPMD Reports 0 Problems

### ISSUE THREAD

**Issue by amccrodan** [sandhje/vscode-phpmd#59]
Hi there,

Having just installed this extension (VS Code 1.50.1 on OSX Mojave 10.14.6), I see it working normally for most files. However, when I open up our repo's most egregious, legacy controller, PHPMD reports 0 issues with the following output:

```
[Info  - 10:39:34 AM] Document saved, starting validation.
[Info  - 10:39:34 AM] PHP Mess Detector validation started for file:///path/to/giant-controller.php
[Info  - 10:39:34 AM] PHP command test successful (PHP 7.1.33 (cli) ...)
[Info  - 10:39:34 AM] Building phpmd version command: php /Users/username/.vscode/extensions/ecodes.vscode-phpmd-1.2.0/phpmd/phpmd.phar --version
[Info  - 10:39:34 AM] PHP Mess Detector test successful (PHPMD 2.6.0)
[Info  - 10:39:34 AM] File /path/to/giant-controller.php test successful
[Info  - 10:39:34 AM] Running phpmd command with options ""/path/to/giant-controller.php" xml "cleancode,codesize,controversial,design,unusedcode,naming""
[Info  - 10:39:34 AM] Building phpmd command: php /Users/username/.vscode/extensions/ecodes.vscode-phpmd-1.2.0/phpmd/phpmd.phar "/path/to/giant-controller.php" xml "cleancode,codesize,controversial,design,unusedcode,naming"
[Info  - 10:39:40 AM] PHP Mess Detector validation completed for file:///path/to/giant-controller.php. 0 problems found
[Info  - 10:39:40 AM] Document validation after save completed successfully
```

If I run the exact same command `php /Users/username/.vscode/extensions/ecodes.vscode-phpmd-1.2.0/phpmd/phpmd.phar "/path/to/giant-controller.php" xml "cleancode,codesize,controversial,design,unusedcode,naming"` on the OSX cli, it spits out ~4000 violations. Any ideas?

**Comment by mikebronner** [comment]
I'm seeing this issue as well when developing using the RemoteSSH plugin.

**Comment by sandhje** [comment]
So far I was unable to reproduce this locally. Is it possible to send me a copy of the PHP file in question so I can try to reproduce this? You can send an email to sandhje-at-ecodes.io.
